packages/scripts/src/ManualRelay.js

```diff
@@ -50,7 +50,7 @@ async function main() {
     Logger,
     l1Web3,
     process.env.BRIDGE_ADMIN_ADDRESS || (await getAddress("BridgeAdmin", 1)),
-    sdk.across.constants.RATE_MODELS
```

Comment:
Is there a reason these constants are still there? Keeping them around seems like an easy way to create bad relays, especially since they're out of date now.

Reply:
@daywiss will remove this when the FE client no longer uses this. Can it be removed now?

Reply:
safe to remove, no longer in use on FE